packages/core/src/app/shipping/ShippingSummary.tsx

```diff
@@ -0,0 +1,55 @@
+
+import { Cart, Consignment } from '@bigcommerce/checkout-sdk';
+import React, { FunctionComponent, memo } from 'react';
+
+import StaticConsignment from './StaticConsignment';
+import StaticMultiConsignment from './StaticMultiConsignment';
+
+
+interface ShippingSummaryProps {
+    isNewMultiShippingUIEnabled: boolean;
+    isMultiShippingMode: boolean;
+    consignments: Consignment[];
+    cart: Cart;
+}
+
+const ShippingSummary: FunctionComponent<ShippingSummaryProps> = ({
+    isNewMultiShippingUIEnabled,
+    isMultiShippingMode,
+    consignments,
+    cart
+}) => {
+    if (isNewMultiShippingUIEnabled && isMultiShippingMode) {
+        return (
+            <>
+                {consignments.map((consignment, index) => (
+                <div className="staticMultiConsignmentContainer" key={consignment.id}>
+                    <StaticMultiConsignment
+                        cart={cart}
+                        consignment={consignment}
+                        consignmentNumber={index + 1}
+                    />
+                </div>
+                ))}
+            </>
```

Comment:
`<>` seens redundant.

```suggestion
                consignments.map((consignment, index) => (
                <div className="staticMultiConsignmentContainer" key={consignment.id}>
                    <StaticMultiConsignment
                        cart={cart}
                        consignment={consignment}
                        consignmentNumber={index + 1}
                    />
                </div>
                ))
```

Reply:
It is for a single parent wrapper. Wouldn't work without it.
<img width="878" alt="image" src="https://github.com/user-attachments/assets/fbaf2c47-0060-47f7-9bd4-63899150c45b">